src/components/ApplicationDetails.vue

```diff
@@ -94,13 +94,15 @@
                         <table class="table table-striped">
                             <thead>
                                 <tr>
-                                    <th>Endpoint</th>
+                                    <th v-if="app.transport_type === 'webengine'">Entry Point Path</th>
```

Comment:
```suggestion
                                    <th v-if="app.transport_type === 'webengine'">Entrypoint Path</th>
```
Entrypoint is one word.